src/components/sections/Hero.section.tsx

```diff
@@ -34,11 +34,11 @@ const HeroSection: React.FC = () => {
     return (
         <div className="bg-midnight text-white py-16">
           <div className="text-center">
-            <h1 className="font-bold text-mint">&lt;HawkHacks 2024/&gt;</h1>           
-            <p className="pb-2 text-xl">Wilfrid Laurier’s first large-scale global hackathon!</p>
-            <p className="pb-3 text-sm font-thin [word-spacing:5px]">Waterloo, ON | April 6th - 8th | In-Person</p>
-            <p className="mx-5 space-x-8 text-base font-bold [word-spacing:45px]">{countdown}</p>
-            <p className="pb-3 text-sm mb-2 font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
+            <h1 className="font-bold text-mint text-7xl">&lt;HawkHacks 2024/&gt;</h1>           
```

Comment:
good fix overall! just try to add a bit spacing at the bottom, seems a bit tight here

Reply:
and then you are good